README.rst

```diff
@@ -154,6 +186,19 @@ A favicon linked to the external URL will be appended to the `Gene Identifier` r
 OMERO.web must be configured with the Django redis cache
 https://docs.openmicroscopy.org/omero/5/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.html?highlight=redis#configuring-omero-web
 which is used to cache the favicons that are obtained using a Google service.
+If your IT structure utilizes a proxy and you are unwilling to set the proxy on a system level for the OMERO(.web) server you can set one directly in the ``requests.get()`` `method <https://github.com/ome/omero-mapr/blob/99dfb1a17418dbc996b9cb402e35db9d8e4b79f8/omero_mapr/views.py#L669>`_ like this
+
+::
+
+    proxies = {'http':'http://wwwproxy.<youproxy>.de:80','https':'http://wwwproxy.<yourproxy>.de:80'}
+        r = requests.get(
+            "%s%s" % (mapr_settings.FAVICON_WEBSERVICE, favdomain),
+            stream=True, proxies = proxies)
+
+
+Map Annotations on Wells/Images
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For Plates the same Map Annotation that is on an Image has to also be on its corresponding Well for the Values to be correctly findable in the search function.
```

Comment:
I don't know if it feels like some explanation is needed for this, but it's partly historical (it used to be the case that we didn't show any Annotations for Images when browsing Screen-Plate-Well data) and partly for performance since it saves a join in the query when you're trying to find matching Plates if you only need to query Wells instead of Images.

Reply:
As a user my intuition would be that a Key-Value pair on an image is searchable independent from where that image is "located".
If it is not - even if it's for a good reason - I think it should be explained how to make it searchable.
And after all it is just one sentence.

Reply:
Sorry - I wasn't very clear there. I *definitely* think that it needs mentioning that "you have to put the KVPs on Images and Wells" as you've done there. I was suggesting that it might also benefit from explaining to users *why* that this behaviour exists (mix of history and performance). It isn't needed, but it might help users to understand this strange requirement.